Sources/PackageDescription/PackageDescription.swift

```diff
@@ -203,27 +189,21 @@ public final class Package {
     /// The list of package dependencies.
     public var dependencies: [Dependency]
 
-  #if PACKAGE_DESCRIPTION_4
-    /// The list of Swift versions that this package is compatible with.
-    public var swiftLanguageVersions: [Int]?
```

Comment:
This is a breaking change, right? In the past, we went through the trouble creating two versions because we didn't want to break this part of the API but I doubt that there are a lot of packages that will actually get affected by this..